cluster/src/test/scala/org/apache/spark/sql/store/BugTest.scala

```diff
@@ -475,5 +480,76 @@ class BugTest extends SnappyFunSuite with BeforeAndAfterAll {
 
   }
 
+  test("Bug SNAP-2890") {
+    snc
+    var serverHostPort2 = TestUtil.startNetServer()
+    var conn = DriverManager.getConnection(s"jdbc:snappydata://$serverHostPort2")
+    var stmt = conn.createStatement()
+    val snappy = snc.snappySession
+    val numCols = 132
+    snappy.conf.set(Property.ColumnBatchSize.name, "256")
+    snappy.sql("drop table if exists test1")
+    val sb = new StringBuilder
+    for(i <- 1 until numCols + 1) {
+      sb.append(s"col$i string,")
+    }
+    sb.deleteCharAt(sb.length -1)
+
+    snappy.sql(s"create table test1 (${sb.toString()}) " +
+      "using column ")
+
+    val params = Array.fill(numCols)('?').mkString(",")
+    val insertStr = s"insert into test1 values (${params})"
+    val ps = conn.prepareStatement(insertStr)
+    for (i <- 0 until 1000) {
+      for (j <- 1 until numCols + 1) {
+        ps.setString(j, j.toString)
+      }
+      ps.executeUpdate()
```

Comment:
better to do batch insert which will be much more efficient and reduce test running time: ps.addBatch(), and ps.executeBatch() outside the loop